seatunnel-connectors/seatunnel-connectors-seatunnel/seatunnel-connector-seatunnel-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/source/KafkaSourceSplitEnumerator.java

```diff
@@ -147,10 +147,10 @@ private Set<KafkaSourceSplit> getTopicInfo() throws ExecutionException, Interrup
 
     private void assignSplit(Collection<Integer> taskIDList) {
```

Comment:
This method assginSplit to task which in parameter. If change like you code, the parameter should be remove because it's not used. But this method will assginSplit to all task every time, it's ok?

Reply:
You are right, I delete this parameter